### PR TITLE
Italian version

### PR DESCRIPTION
**Pull request by zerbfra**
This is a working in progress PR for the italian release

### CODE REVIEW

**Comment by fossecode** [comment]
- [ ] `config.json` ready
- [x] `app/locales/it-word-list.txt`
- [x] `app/locales/it-municipalities.json`
- [x] `app/locales/it-postalCodeCoordinates.json`
- [x] Add translations to `app/locales/it.json`
- [x] Write a privacy statement in`app/views/privacy-statements/it-lang-privacy-statement.ejs`
- [x] Configure URL paths in `app/domain/urls.ts`
- [x] Domain: `coronastatus.it`

File: app/locales/en.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
...
   "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to": "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to",
-  "this article": "this article",
-  "as an attempt to chart these numbers.": "as an attempt to chart these numbers.",
+  "this article": "Si è preso spunto da questo articolo norgevese",
```
Comment (fossecode) [inline app/locales/en.json]: Woops, this should probably be reverted

Comment (zerbfra) [reply app/locales/en.json]: it is correct: 

```
  "this article": "da questo articolo norgevese",
  "as an attempt to chart these numbers.": "e la piattaforma si è iniziato a sviluppare per raccogliere questi dati.",
```

we changed a little bit the italian translation to better fit our "italian" minds ;) 
here we say that the platform has been inspired by the norwegian article

**Comment by zerbfra** [comment]
Now it should be fine to be merged

File: app/locales/en.json (hunk 1 above)
Comment (fossecode) [reply app/locales/en.json]: Yes, but this is in the english locale

Comment (zerbfra) [reply app/locales/en.json]: should be fixed

**Review by fossecode — APPROVED** [review]
Belissimo!

**Pull request merged**